pkg/agent/testdata/AKSUbuntu1604+Containerd/line105.sh

```diff
@@ -10,6 +10,7 @@ RestartSec=2
 EnvironmentFile=/etc/default/kubelet
 SuccessExitStatus=143
 ExecStartPre=/bin/bash /opt/azure/containers/kubelet.sh
+ExecStartPre=/bin/bash /opt/azure/containers/ensure_imds_restriction.sh
```

Comment:
it looks like, but just wanted to confirm that `ensure_imds_restriction.sh` is idempotent - specifying it as an ExecStartPre means that _any_ time the kubelet service is started/restarted, the script will be executed

Reply:
yes, that's indeed idempotent, before inserting/deleting, it will firstly check whether the rule is already there/already deleted. So executing it multiple times will get the same result.